libs/mutantschemata/cpp_source/rewrite.hpp

```diff
@@ -268,7 +275,7 @@ void writeChangedFiles(bool inPlace) {
         for (std::map<clang::FileID, clang::RewriteBuffer>::iterator I = rewriter.buffer_begin(), E = rewriter.buffer_end(); I != E; ++I) {
             const clang::FileEntry *file = rewriter.getSourceMgr().getFileEntryForID(I->first);
             if (file) {
-                if (file->isValid()) {
+	      if (file->isValid()) {
```

Comment:
My emacs is completely bonkers with the indentations. Certainly something we solve later with format-checkers.